ert_gui/tools/plot/plot_api.py

```diff
@@ -1,12 +1,28 @@
 from ert_data import loader as loader
 import pandas as pd
 from ert_data.measured import MeasuredData
+from typing import List
+import requests
+from ert_storage.client import Session
 
 
 class PlotApi(object):
     def __init__(self, facade):
         self._facade = facade
 
+    def _getExperiments(self) -> List[str]:
```

Comment:
Is there a special reason this function is not called? `_get_experiments` ?

Reply:
No. I will change it